sycl/doc/developer/DockerBKMs.md

```diff
@@ -143,10 +143,20 @@ instructions.
 
 ## Changing Docker user
 
-By default all processes inside Docker run as root. Some LLVM or Clang tests
-expect your user to be anything but root. You can change the user by specifying
-`-u <username or uid>` option. All Docker containers come with user `sycl`
-created.
+By default all processes within our containers are run as the `sycl_ci` user.
+Note: it **does not** have password-less `root` access.
+
+If you want to change the user, you can do that by specifying the
+`-u <username or uid>` option when running the container.
+
+All containers come with the `/user-setup.sh` script which can used to create
+the `sycl` user which has all the same groups as the `sycl_ci` user, but also
+has password-less access to `root`. Use the script as follows:
+
+```bash
+# Note: the script requires root permissions to create a new user
+/user-setup.sh --regular
+```
```

Comment:
@AlexeySachkov, can you please give an example of commands to switch the user to `sycl`, please? I'm not sure I understand what commands to use from this text.

Reply:
Expanded this code snippet in db221e4a7007d77d27e3355b7ac72a74fbf531e8